Typescript type error for GraphQLFieldExtensions

There is a wrong typing for `GraphQLFieldExtensions` which is not compatible with the current graphql package leading to a TypeScript error.

join-monster: 3.3.2
graphql: 16.8.1

```
node_modules/graphql/type/definition.d.ts:526:18 - error TS2428: All declarations of 'GraphQLFieldExtensions' must have identical type parameters.

526 export interface GraphQLFieldExtensions<_TSource, _TContext, _TArgs = any> {
                     ~~~~~~~~~~~~~~~~~~~~~~

node_modules/join-monster/dist/index.d.ts:117:13 - error TS2428: All declarations of 'GraphQLFieldExtensions' must have identical type parameters.

117   interface GraphQLFieldExtensions<
                ~~~~~~~~~~~~~~~~~~~~~~
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "join-monster",
-  "version": "3.3.2",
+  "version": "3.3.3",
   "description": "A GraphQL to SQL query execution layer for batch data fetching.",
   "main": "dist/index.js",
   "engines": {

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -117,7 +117,7 @@ declare module 'graphql' {
   interface GraphQLFieldExtensions<
     _TSource,
     _TContext,
-    _TArgs = { [argName: string]: any }
+    _TArgs = any
   > {
     joinMonster?: FieldConfigExtension<_TSource, _TContext, _TArgs>
   }

